tensorhive/api/APIServer.py

```diff
@@ -3,6 +3,9 @@
 from tensorhive.config import API_CONFIG
 from tensorhive.database import db_session
 from flask_cors import CORS
+from specsynthase.specbuilder import SpecBuilder
```

Comment:
These three imports seem unused. Also, no corresponding dependencies in setup.

Reply:
Fix in b499874eb9fab4e1fcab80803522b065e733f42b